### PR TITLE
Kk login overhaul

### PR DESCRIPTION
**Pull request by katiekeel**

### CODE REVIEW

File: app/controllers/api/v1/player_points_controller.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 class Api::V1::PlayerPointsController < ApplicationController
-  skip_before_action :authorize!
```
Comment (mdevoe12) [inline app/controllers/api/v1/player_points_controller.rb:2]: @katiekeel 

With this stripped out, does devise take care of authorization under the hood?

Comment (katiekeel) [reply app/controllers/api/v1/player_points_controller.rb:2]: Not really. Current_user is still the same as it was. So we can still use the same checks we wrote before that uses it. I fixed the authorization tests that failed so we should be ok 👍

Comment (mdevoe12) [reply app/controllers/api/v1/player_points_controller.rb:2]: Sounds good. Merging.

🌮

**Review by mdevoe12 — COMMENTED** [review]
Comment on ApplicationController regarding authorize! method

**Review by mdevoe12 — APPROVED** [review]

**Pull request merged**